README.md

```diff
@@ -1,57 +1,355 @@
-# Project Name
+---
+page_type: sample
+languages:
+  - python
+  - django
```

Comment:
"django" would be a product, but it's not in [taxonomy](https://review.docs.microsoft.com/en-us/new-hope/information-architecture/metadata/taxonomies?branch=master)